src/test/java/hw2/ex1/FirstScenarioTest.java

```diff
@@ -0,0 +1,203 @@
+        // comparing it with expected title -- "Home page"
+        String actualTitle = driver.getTitle();
+        assertEquals(actualTitle, "Home Page");
+
+        // 3. Perform login
+        // There we have a simple login steps:
+        // - clicking on icon
+        // - entering username and password
+        // - clicking on a login-button
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        // 4. Assert User name in the left top side
+        // of screen that user is logined
+        // There we have a getting of username and
+        // comparing it with expected -- "PITER CHAILOVSKII"
+        // (We should compare it as is - even if it's
+        // written in upper case
+        WebElement actualUserName = driver.findElement(By.id("user-name"));
+        assertEquals(actualUserName.getText(), "PITER CHAILOVSKII");
+
+        // 5. Assert Browser title
+        // There's a getting of a title and
+        // comparing it with expected title -- "Home page"
+        // NB! We take expectedTitle from above
+        actualTitle = driver.getTitle();
+        assertEquals(actualTitle, "Home Page");
+
+        // 6. Assert that there are 4 items on the header
+        // section are displayed and they have proper texts
+        // There's a creating of array list with expected texts
+        // in header, getting a web element with actual header
+        // texts, getting list of these texts and comparing them
+        ArrayList<String> expectedHeaderTexts = new ArrayList<>();
+        expectedHeaderTexts.add("HOME");
+        expectedHeaderTexts.add("CONTACT FORM");
+        expectedHeaderTexts.add("SERVICE");
+        expectedHeaderTexts.add("METALS & COLORS");
+        WebElement header = driver.findElement(By.
+                xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']"));
+        List<String> actualHeaderTexts = Arrays.
+                asList(header.getText().split("\n"));
+        assertEquals(actualHeaderTexts, expectedHeaderTexts);
+
+        // 7. Assert that there are 4 images on the Index Page and
+        // they are displayed
+        // There's a simple assertion of existing
+        // and displaying of 4 icons
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-practise']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-custom']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-multi']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-base']")).
+                isDisplayed());
```

Comment:
These set of assertions could be simplified)

Reply:
solved: I collect the list of these benefit-icons and assert them in for-each loop